doc/api/errors.md

```diff
@@ -35,7 +35,7 @@ are handled using the [`try…catch` construct][try-catch] provided by the
 JavaScript language.
 
 ```js
-// Throws with a ReferenceError because z is undefined
+// Throws with a ReferenceError because z is not defined
```

Comment:
This is fine as it's an improvement over what's there, but two totally insignificant suggestions to consider:

* Put a period at the end of the comment as it's a full sentence.
* Maybe instead of "not defined", it should be "undeclared" or even "non-existent"? Those are the terms I see used in the MDN article on ReferenceError, so might as well steal from them. (I looked in the spec too but I don't think the language there would be helpful here, at least not as far as I saw.)

Reply:
@Trott 
* for the period I've added it
* I think `not defined` will be closer to the error thrown which will be `ReferenceError: z is not defined`.

Reply:
(The period isn't showing up in the GitHub interface. Do you need to  push or force-push to your origin maybe?)

Reply:
Sorry for that, It's now here.